File: Lei_Garay_Code/Homework01/src/com/revature/question06/IsEven.java
package com.revature.question06;

public class IsEven {
	
	static boolean isEven(int num)
	{
		double numFloor = Math.floor( (double) num / 2);
		
		if ( ( (double) num / 2 - numFloor == 0))
		{
			System.out.println("true");
			return true;
		}
		System.out.println("false");
		return false;
	}
	
	public static void main(String[] args) {
		isEven(2);
		isEven(191);
	}
}

File: Edwin_Faican_Code/Week 3/JavaScript Assisgment/js/iseven.js
const isEven = function(n) {
    let num = Math.floor(n/2);
    if(num*2 == n) {
        return 'true';
    } else {
        return 'false';
    }
}

function even() {
    $('#reseven').html(isEven($('#ineven').val()));
}